.github/workflows/publish.yml

```diff
@@ -52,6 +52,31 @@ jobs:
         env:
           AWS_ACCESS_KEY_ID: ${{ secrets.AWS_ACCESS_KEY_ID }}
           AWS_SECRET_ACCESS_KEY: ${{ secrets.AWS_SECRET_ACCESS_KEY }}
+      -
+        name: Update redirects
+        run: |
+          path_join() {
+            part1=$1
+            part2=$2
+            if [ "${part1: -1}" == "/" ]; then
+              part1=${part1}${part2}
+            else
+              part1=${part1}/${part2}
+            fi
+            echo $part1
+          }
+          for redirect in $(cat ./_site/redirects.json | jq -c '. | to_entries | .[]'); do
```

Comment:
I think the `cat` may not be needed; `jq` accepts filename(s) as positional argument; so something like this should work;

```bash
jq -c '. | to_entries | .[]' ./_site/redirects.jso
```